pydeepmerge/_merge_configs.py

```diff
@@ -0,0 +1,72 @@
+from pathlib import Path
+
+from pydeepmerge import deep_merge
+from pydeepmerge.errors import (
+    FileTypeError,
+    ParserNotAvailableError
+)
+from pydeepmerge.parsers import load_parsers
+from pydeepmerge.strategies import prefer_right
+from pydeepmerge.types import MergeStrategy
+from typing import (
+    Dict,
+    List,
+)
+
+
+def merge_configs(
+    *files: List[str],
+    merge_strategy: MergeStrategy = prefer_right,
+    auto: bool = True,
+    strict: bool = True,
+    extra_parsers: Dict = {}
+) -> Dict:
+    '''
+    A function that takes a sequence of config filenames and produces
+    a dictionary resulting from all the files being processed and then merged
+    with a specified merge strategy.
+
+    To use custom parsers, pass a flat dictionary of filetypes and parsers.
+    If `auto` is set to False, use only the parsers explicitly passed
+    through the `extra_parsers` parameter.
+
+    If `strict` is set to True, all file names must have the same extension.
+    This also applies to file type variations like `.yml` vs `.yaml`
+    '''
+    parsers = {}
```

Comment:
Nit: I would move this down, or the parser assignments up to keep related code together